tests/pages/dashboardPage.js

```diff
@@ -691,23 +694,28 @@ module.exports = {
   },
 
   fields: {
+    annotationMarker: '(//div[contains(@class,"events_marker")])',
+    clearSelection: '//a[@ng-click="vm.clearSelections()"]',
+    collapsedDashboardRow: '//div[@class="dashboard-row dashboard-row--collapsed"]/a',
+    dataLinkForRoot: '//div[contains(text(), "Data links")]/..//a',
+    Last2Days: '//span[contains(text(), "Last 2 days")]',
```

Comment:
any reason why locate can not be used here?

Reply:
I understand you are just changing the position, but you could also improve the locator as you deal with them.

Reply:
@IvanaBrecherova do you think they are organized alphabetically?

Reply:
@puneet0191 Fixed. Thanks for notice.